page/docs/intro.md

```diff
@@ -19,7 +19,7 @@ Let's discover **[starknet-devnet](https://github.com/Shard-Labs/starknet-devnet
 
 ## Requirements
 
-Works with Python versions >=3.8 and <3.10.
+Works with Python versions >=3.8 and <3.10 and also needs Cairo-rs-py source code. 
```

Comment:
I documented this in `page/docs/run.md` (paragraph `Run with the Rust implementation of Cairo VM`) since using Rust VM (at least in the next Devnet release) will only be optional. So having cairo-rs-py will not be a prerequisite for the default running.

I will merge this PR and change this sentence myself.